usr/sbin/so-clear-backlog

```diff
@@ -71,40 +71,45 @@ if [ $SKIP -ne 1 ] ; then
         if [ "$INPUT" != "YES" ] ; then exit 0; fi
 fi
 
+ENGINE=${ENGINE:1}
```

Comment:
This is to handle the suricata/Suricata issue, right?  I'd prefer to use the same strategy that we used in rule-update:
https://github.com/Security-Onion-Solutions/securityonion-rule-update/blob/master/usr/bin/rule-update#L340-L343